helm/templates/dcgm-exporter-daemonset.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: dcgm-exporter
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: dcgm-exporter
+    version: v1
+spec:
+  selector:
+    matchLabels:
+      k8s-app: dcgm-exporter
+  template:
+    metadata:
+      labels:
+        k8s-app: dcgm-exporter
+        version: v1
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccountName: {{ template "cloudwatch-agent.serviceAccountName" . }}
```

Comment:
should we make a different service account for the dcgm exporter instead of using the cloudwatch agent service account? There might be different privileges

Reply:
Yeah, great call. updated